be/src/runtime/runtime_state.h

```diff
@@ -434,12 +434,31 @@ class RuntimeState {
                _query_options.enable_collect_table_level_scan_stats;
     }
 
-    bool is_jit_enabled() const;
-
     bool enable_wait_dependent_event() const {
         return _query_options.__isset.enable_wait_dependent_event && _query_options.enable_wait_dependent_event;
     }
 
+    bool is_jit_enabled() const;
+
+    bool is_adaptive_jit() const { return _query_options.__isset.jit_level && _query_options.jit_level == 1; }
+
+    void set_jit_level(const int level) { _query_options.__set_jit_level(level); }
+
+    // except /, %
+    bool is_jit_arithmetic_op() const { return (_query_options.jit_level == 1) || ((_query_options.jit_level & 2)); }
```

Comment:
better to define an enum type for these constant variables? like the following:
```cpp
enum CompilableOperatorType : uint8_t {
    ALL = 1,
    ARITHMETIC = 2
   ...
};
```

Reply:
it would be unnecessary.

Reply:
suggest to put which expr support jit in one function, is_jit_arithmetic_op seems puzzling